pkg/cmd/server/kubernetes/node.go

```diff
@@ -280,39 +283,43 @@ func (c *NodeConfig) RunServiceStores(enableProxy, enableDNS bool) {
 
 // RunKubelet starts the Kubelet.
 func (c *NodeConfig) RunKubelet() {
-	if c.KubeletConfig.ClusterDNS == nil {
+	var clusterDNS net.IP
+	if c.KubeletServer.ClusterDNS == "" {
```

Comment:
hahahahaha.  Really?  `len(c.KubeletServer.ClusterDNS)`.  That's funny from you.